### DISCUSSION TITLE
center align with pm/am

### DISCUSSION THREAD

**Discussion by ezra717** [bluecaret/carettab#326] [Ideas and Feature Requests]
hello
nice software. thanks 
I am trying to use this with Opera browser and failed to find custom url to set as start page instead of it's own speeddial.

btw

if you can give me an option to align center 03:32:11am as same as 15:32:11 or 03:32:11 without am/pm that would be nice. 
I want to align time in center and am/pm attached, not the whole thing in center.

easiest way to do it is put 2 spaces in front of hour, or am/pm same as background color in front of hour
like this 
am03:32:11am 
of course first am should not be showing (same as background color). 
or maybe you have different programming solution.

thank you

**Comment by bluecaret** [comment]
This will be in the next update. Cheers!
